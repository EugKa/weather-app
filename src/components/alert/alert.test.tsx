import React from 'react';
import { shallow } from "enzyme";
import { CustomAlert } from "./index";

const props = {
    type: 'error',   
    children: 'Error'   
}

const setUp = (props:any) => shallow(<CustomAlert {...props}/>)

describe('CityCard component', () => {    
    it("should render CityCard component with propsData", () => {
        const component = setUp(props)
        expect(component).toMatchSnapshot()
    }) 
})