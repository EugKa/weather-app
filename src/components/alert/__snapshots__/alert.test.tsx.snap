// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityCard component should render CityCard component with propsData 1`] = `
<ForwardRef(Stack)
  spacing={2}
  sx={
    Object {
      "width": "100%",
    }
  }
>
  <ForwardRef(Alert)
    severity="error"
  >
    Error
  </ForwardRef(Alert)>
</ForwardRef(Stack)>
`;
